app/views/pages/home.html.haml

```diff
@@ -26,25 +26,24 @@
     %section.content-section
       .grid-row
         .column-one-half
-          %h2.heading-medium.content-section__title Use and share government data
-          %p GOV.UK Registers helps you to:
-          %ul.list.list-bullet
-            %li build and maintain your service with accurate data that is updated for you
-            %li reduce the time and cost of sourcing data from across government
-            %li easily share data with other government departments
+          %h2.heading-medium.content-section__title What is a register?
+          %p A register is a list of information that is managed and approved by government.
+          %p For example, the country register lists all countries recognised by the Foreign and Commonwealth Office.
+          %p You can use registers to build your service.
+          %p The UK Government and Parliament petitions service uses the country register to help users select their country from a dropdown menu, for instance.
         .column-one-half
           = image_tag 'get-data.png'
 
     %section.content-section.content-section--with-top-border
       .grid-row
         .column-one-half
-          %h2.heading-medium.content-section__title How registers work
-          %p Registers are live lists of information that are managed and approved by government. They are the most reliable lists of their kind.
-          %p Each register:
+          %h2.heading-medium.content-section__title Trusted by government services
+          %p By using registers you can: 
           %ul.list.list-bullet
-            %li can be accessed and used freely by anyone
-            %li provides data in different formats including JSON and CSV
-            %li has a full version history to ensure transparency of data changes
+            %li reduce the time and cost of sourcing data from across government
+            %li be confident that your service is using the most up-to-date government data 
```

Comment:
On desktop view, 'up-to-date' is split across two lines. Can we put a break in so 'up-to-date' sits on the next line (without messing up the mobile view)?

Reply:
Now changed to use non-breaking hyphens.